### DISCUSSION TITLE
Best way to use Pure with multiple computers?

### DISCUSSION THREAD

**Discussion by plainspace** [sindresorhus/pure#593] [General]
Sorry if this has been asked before. I searched the issues and didn't find anything relevant. I use multiple computers and I'd like to use Pure and sync things like aliases across machines. Any tips?

**Comment by mafredri** [comment]
I think if you ask 10 people, you'd find 10 "best" ways to do this 😄.

My personal approach to it is [mafredri/zeet](https://github.com/mafredri/zeet). It contains all my zsh configuration (including the Pure theme). I checkout the project on every machine I use, it has auto-updating built in so I just need to push updates to the git repository and updates will be automatically pulled when new shells are opened on the other machines.

**Reply by 0x07CB** [reply]
I love that but now my computer os dead... I try to repair that this night with an complete chroot ... Yeah I love ArchLinux too I am Rick Sanchez !!!!
